Mathlib/RingTheory/DedekindDomain/AdicValuation.lean

```diff
@@ -278,6 +278,13 @@ theorem intValuation_singleton {r : R} (hr : r ≠ 0) (hv : v.asIdeal = Ideal.sp
   rw [v.intValuation_if_neg hr, ← hv, Associates.count_self, Int.ofNat_one]
   exact v.associates_irreducible
 
+@[simp]
+theorem intValuation_eq_one_iff {R : Type*} [CommRing R] [IsDedekindDomain R]
```

Comment:
Could you reuse the `variable`s `{R : Type*} [CommRing R] [IsDedekindDomain R]`?